source/extensions/filters/http/cache/http_cache.h

```diff
@@ -39,17 +39,14 @@ struct LookupResult {
   // header with this value, replacing any preexisting content-length header.
   // (This lets us dechunk responses as we insert them, then later serve them
   // with a content-length header.)
+  // May be 0 if the cache entry is streaming.
```

Comment:
Fair, I was thinking of doing that as a next step since it's also a prerequisite of the streaming behavior to *know* whether you know. It's a pretty hefty step since this content length is used in a bunch of places, which is why I was going to do it separately.